client/client-sdk-javascript/binding.gyp

```diff
@@ -18,8 +18,8 @@
             "conditions": [
                 ["OS=='mac'", {
                     "libraries": [
-                        "-Wl,-rpath,<(module_root_dir)/deps/crypto-sdk/build/mac/lib/",
-                        "<(module_root_dir)/deps/crypto-sdk/build/mac/lib/libcryptosdk.dylib"
```

Comment:
Actually, you're right. I wanted to standardize it, since it had mixed case in a couple of places, but lowercase letters is more idiosyncratic. I'll fix it in this PR very soon.